web/client/utils/QueryParamsUtils.js

```diff
@@ -0,0 +1,52 @@
+import url from "url";
+import {get} from "lodash";
+
+/**
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const getRequestLoadValue = (name, state) => {
```

Comment:
Document this function (please add parameters and correct if wrong): 
```suggestion
/**
 * Retrieves parameters from hash "query string" of react router
 * Example: `#/viewer/openlayers/0?center=0,0&zoom=5
 */
export const getRequestLoadValue = (name, state) => {
```